lib/ui/views/stop_search_view.dart

```diff
@@ -40,7 +40,6 @@ class _SearchStopsState extends State<SearchStops> {
           Expanded(
               child: StopQueryResults(
             searchName: searchName,
-            maxResults: 10,
```

Comment:
Bad things seem to happen without this limit (making the name field empty freezes the application)